app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,28 @@
+
+
+<!-- <h1>TaskList</h1>
+<p>Keeping Track of To-do's</p> -->
+
+<ul>
+  <% @tasks.each do |task| %>
+  <li>
+
+    <input type="checkbox" <%="checked" if task.completed? %> disabled >
+
```

Comment:
I like this checkbox. This is a clever way to indicate whether the task is done.